documentation/asciidoc/topics/proc_installing_command_line.adoc

```diff
@@ -0,0 +1,66 @@
+[id='installing-openshift-command-line_{context}']
+= Installing the Helm chart on the command line
+
+[role="_abstract"]
+Use the `helm` and `oc` clients to install the {brandname} Helm chart from the Red Hat Helm Charts repository via the command line.
+ifdef::community[]
+This approach lets you automate the deployment of {brandname} clusters on {openshiftshort} or to install the {brandname} Helm chart on Kubernetes clusters.
+endif::community[]
+ifdef::downstream[]
+This approach lets you script the deployment of {brandname} clusters on {openshiftshort}.
+endif::downstream[]
+
+.Prerequisites
+
+ifdef::community[]
+* Have access to an {openshiftshort} or Kubernetes cluster.
+* Have an `oc` or `kubectl` client.
+endif::community[]
+ifdef::downstream[]
+* Have access to an {openshiftshort} cluster.
+* Have an `oc` client.
+endif::downstream[]
+* Have a `helm` client.
```

Comment:
Is "Have" necessary on each of these points?

Reply:
No, I'll fix that. Agreed that it's a bit cluttered.